src/synapse/workload.py

```diff
@@ -137,6 +139,23 @@ def check_mjolnir_ready() -> ops.pebble.CheckDict:
     return check.to_dict()
 
 
+def create_well_know_file(container: ops.Container, charm_state: CharmState) -> None:
```

Comment:
This function seems to generate a specific well known file (`.well-known/matrix/server`) compared to the function name `create_well_know_file` which seems to read like it can create a range of well known files.

Do you think this should be renamed to something with less implementation detail? Like create configure server name or something like that? The fact that it is creating a well known file seems to be an implementation detail the caller probably doesn't need to know